src/main/java/edu/kis/powp/jobs2d/drivers/RealTimeDecoratorDriver.java

```diff
@@ -0,0 +1,67 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import javax.swing.*;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class RealTimeDecoratorDriver implements Job2dDriver {
+    private JPanel panel;
+    private Queue<Runnable> tasks = new LinkedList<>();
+    private final int INTERVAL_MS = 300;
```

Comment:
There should be no MAGIC numbers. Please use different units, i.e., velocity (e.g. length of line per second). It should be configured separately for setPosition and operateTo operations. Refresh rate (for operateTo) would be also nice ;).